### PR TITLE
Update transfer nugets

### PR DESCRIPTION
**Pull request by sahma19**
This PR is part of the ZenHub Issue:

https://app.zenhub.com/workspaces/team-atlas-633199659e255a37cd1d144f/issues/gh/energinet-datahub/energy-origin-issues/1868

### CODE REVIEW

File: domains/transfer/Transfer.API/API.IntegrationTests/DbContextExtension.cs [hunk 1]
```diff
@@ -33,10 +33,13 @@ public static async Task<List<T>> RepeatedlyQueryUntilCountIsMet<T>(this DbConte
...
+        var historyEntriesTable = dbContext.Model.FindEntityType(typeof(TransferAgreementHistoryEntry))!.GetTableName();
 
-        await dbContext.Database.ExecuteSqlRawAsync($"TRUNCATE TABLE \"{agreementsTable}\" CASCADE");
-        await dbContext.Database.ExecuteSqlRawAsync($"TRUNCATE TABLE \"{historyTable}\"");
+        var transfersSql = $"TRUNCATE TABLE \"{transfersTable}\" CASCADE";
+        var historyEntriesSql = $"TRUNCATE TABLE \"{historyEntriesTable}\"";
+
+        await dbContext.Database.ExecuteSqlRawAsync(transfersSql);
+        await dbContext.Database.ExecuteSqlRawAsync(historyEntriesSql);
     }
```
Comment (tnickelsen) [inline domains/transfer/Transfer.API/API.IntegrationTests/DbContextExtension.cs:44]: Why this change?

Comment (sahma19) [reply domains/transfer/Transfer.API/API.IntegrationTests/DbContextExtension.cs:44]: After upgrading from ef-core 7 to ef-core 8, the linter is complaining about how we execute raw sql statements, asking us to make it safe, from SQL injection. I refactored it, so it isn't complaining anymore, and so it is safe.

**Review by tnickelsen — APPROVED** [review]
